changes/v4.4.18-en.md

```diff
@@ -0,0 +1,10 @@
+# v4.4.18
+
+## Enhancements
+
+- Adds a new feature to enable partial certificate chain validation for TLS listeners[#10553](https://github.com/emqx/emqx/pull/10553).
+  If TLS listener has `partial_chain` set to `cacert_from_cacertfile`,
```

Comment:
maybe we can make it a boolean config?
and:
"if partial_chain is set to 'true', the last certificate in cacertfile is treated as the terminal of the certificate trust-chain. That is, the client may not have to send the full chain in TLS handshake, and EMQX will not try to validate the chain all the way up to the root CA".

Reply:
Set to `true` is a bit vague, value `cacert_from_cacertfile` is more clear about validation composes a trust chain with the cacert in the `cacertfile` not from the cert chain from the client.

I will keep the value `cacert_from_cacertfile ` and map `true` to it.